src/Faithlife.WebRequests/Json/JsonWebServiceClientBase.cs

```diff
@@ -130,6 +158,20 @@ private Uri DoGetRequestUri(string relativeUriPattern, IEnumerable<KeyValuePair<
 			return uri;
 		}
 
+		private Uri DoGetRequestUri(string relativeUriPattern, params string[] parameters)
+		{
+			string uriText = m_baseUri.AbsoluteUri;
+
+			if (!string.IsNullOrEmpty(relativeUriPattern))
+				uriText = uriText.TrimEnd('/') + "/" + relativeUriPattern.TrimStart('/');
```

Comment:
The logic to this point is duplicated above and should be extracted into a method.

Reply:
Counterpoint: https://faithlife.com/posts/1145458

:trollface: